packages/syft/src/syft/core/node/common/node.py

```diff
@@ -383,7 +384,8 @@ def get_metadata_for_client(self) -> Metadata:
             version=str(__version__),
         )
 
-    def get_api(self) -> SyftAPI:
+    def get_api(self, signing_key: Optional[SyftSigningKey] = None) -> SyftAPI:
```

Comment:
MAYBE: Lets make the signing_key a mandatory argument, so we could enforce them to always pass the signing key of the user requesting the api.?